knative-operator/deploy/resources/kourier/kourier-latest-mesh.yaml

```diff
@@ -0,0 +1,346 @@
+apiVersion: v1
```

Comment:
Is there anything special in this file or why is it suffixed with "-mesh"?

Reply:
Merged -mesh manifest into -debug.yaml.